Sources/MessagingInApp/Views/InAppMessageView.swift

```diff
@@ -65,5 +71,36 @@ public class InAppMessageView: UIView {
         //      }
     }
 
-    private func displayInAppMessage(_ message: Message) {}
+    private func displayInAppMessage(_ message: Message) {
```

Comment:
This function's body is the logic that creates a WebView and renders the in-app message in the WebView. Most of this logic comes from `InlineMessageManager`. So all we need to do in the inline View is create a `InlineMessageManager` instance and setup the WebView for displaying in this View.